test/scripts/measures/2017/import-qcdr-measures-spec.js

```diff
@@ -7,29 +7,30 @@ const path = require('path');
 const _ = require('lodash');
 
 // Test data
-const testMeasures = '../../test/scripts/measures/fixtures/test-measures-data.json';
-const testMeasures2 = '../../test/scripts/measures/fixtures/test-measures-data2.json';
-const testCsv = '../../test/scripts/measures/fixtures/test-qcdr.csv';
-const testCsv2Cols = '../../test/scripts/measures/fixtures/test-qcdr-2cols.csv';
-const qcdrStrataNamesFile = '../../util/measures/qcdr-measures-strata-names.json';
-const outputArg = '../../test/scripts/measures/fixtures/test-measures-data-output.json';
-const output = '../' + outputArg;
+const year = 2017;
```

Comment:
Same note here about the really weak preference

Reply:
(see prev comment)